x-pack/filebeat/docs/inputs/input-cel.asciidoc

```diff
@@ -1,876 +0,0 @@
-    "header": <HTTP response headers if a network request>,
-    "rate_limit": <HTTP rate limit map if required by API>, <3>
-    "want_more": false <4>
-}
-----
-
-<1> The `events` field must be present, but may be empty or null. If it is not empty, it must only have objects as elements.
-The field should be an array, but in the case of an error condition in the CEL program it is acceptable to return a single object instead of an array; this will will be wrapped as an array for publication and an error will be logged. If the single object contains a key, "error", the error value will be used to update the status of the input to report to Elastic Agent. This can be used to more rapidly respond to API failures.
-
-<2> If `cursor` is present it must be either be a single object or an array with the same length as events; each element _i_ of the `cursor` will be the details for obtaining the events at and beyond event _i_ in the `events` array. If the `cursor` is a single object it is will be the details for obtaining events after the last event in the `events` array and will only be retained on successful publication of all the events in the `events` array.
-
-<3> If `rate_limit` is present it must be a map with numeric fields `rate` and `burst`. The `rate_limit` field may also have a string `error` field and other fields which will be logged. If it has an `error` field, the `rate` and `burst` will not be used to set rate limit behavior. The {mito_docs}/lib#Limit[Limit], and {mito_docs}/lib#OktaRateLimit[Okta Rate Limit policy] and {mito_docs}/lib#DraftRateLimit[Draft Rate Limit policy] documentation show how to construct this field.
-
-<4> The evaluation is repeated with the new state, after removing the events field, if the "want_more" field is present and true, and a non-zero events array is returned. If the "want_more" field is present after a failed evaluation, it is set to false.
-
-The `status_code`, `header` and `rate_limit` values may be omitted if the program is not interacting with an HTTP API end-point and so will not be needed to contribute to program control.
-
-==== Debug state logging
-
-The CEL input will log the complete state after evaluation when logging at the DEBUG level.
-This will include any sensitive or secret information kept in the `state` object, and so DEBUG level logging should not be used in production when sensitive information is retained in the `state` object.
-See <<cel-state-redact,`redact`>> configuration parameters for settings to exclude sensitive fields from DEBUG logs.
-
-
-==== CEL extension libraries
-
-As noted above the `cel` input provides functions, macros, and global variables to extend the language.
-
-* {mito_docs}/lib#Collections[Collections]
-** {mito_docs}/lib#hdr-Collate[Collate]
-** {mito_docs}/lib#hdr-Drop[Drop]
-** {mito_docs}/lib#hdr-Drop_Empty[Drop Empty]
-** {mito_docs}/lib#hdr-Flatten[Flatten]
-** {mito_docs}/lib#hdr-Front[Front]
```

Comment:
This line is missing (see [here](https://github.com/elastic/beats/pull/42897/files#diff-787354525c35f1b019d662b6e290dcbb2daa31b758d421990718a0055599a760R141-R142)).